src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+const populationSpans = document.querySelectorAll('span.population');
+const totalPopSpan = document.querySelector('span.total-population');
+const averagePopSpan = document.querySelector('span.average-population');
+
+const totalPopulation = [ ...populationSpans ]
+  .map(span => Number(span.innerHTML.split(',').join('')) || 0)
+  .reduce((sum, population) => sum + population);
+
+const averagelPopulation = totalPopulation / populationSpans.length;
+
+totalPopSpan.innerHTML = totalPopulation.toLocaleString();
```

Comment:
look to the file  README.md, point 5 - (same numeric style as populations presented), better to change this)

Reply:
![image](https://user-images.githubusercontent.com/117474961/210213314-a77cd55d-920b-4fe3-9936-ebf25972c18e.png)